src/main/services/steam.service.ts

```diff
@@ -1,11 +1,13 @@
 import { UtilsService } from "./utils.service";
 import regedit from 'regedit'
-import path from "path";
+import { path, join } from "path";
 import { parse } from "@node-steam/vdf";
 import { readFile } from "fs/promises";
 import { spawn } from "child_process";
 import { pathExist } from "../helpers/fs.helpers";
 import log from "electron-log";
+import psList from 'ps-list';
+import os from 'os';
```

Comment:
os and ps-list has not added in package.json and not needed in this branch I believe